site/docs/spark-procedures.md

```diff
@@ -190,6 +190,7 @@ the `expire_snapshots` procedure will never remove files which are still require
 | `table`       | ✔️  | string | Name of the table to update |
 | `older_than`  | ️   | timestamp | Timestamp before which snapshots will be removed (Default: 5 days ago) |
 | `retain_last` |    | int       | Number of ancestor snapshots to preserve regardless of `older_than` (defaults to 1) |
+| `max_concurrent_deletes` |    | int       | Size of the thread pool used for delete file actions (defaults to null, which deletes files serially in the current thread without instantiating a dedicated thread pool) |
```

Comment:
The description is a bit long. How about "by default, no threadpool is used"

Reply:
seconded, reads better to me personally

Reply:
Might be not accurate to say no thread pool at all is used, wonder if we still need to specify 'by default, no dedicated thread pool is used'.  (Not sure if that's too academic).

Reply:
Agreed it's a bit long. Will update.

Reply:
This is updated.